sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -612,6 +616,10 @@ public String getVaultUrl() {
                         return Mono.defer(() -> Mono.just(new PollResponse<>(LongRunningOperationStatus.IN_PROGRESS,
                             pollingContext.getLatestResponse().getValue())));
                     }
+                    if (certificateResponse.getStatusCode() == HttpURLConnection.HTTP_FORBIDDEN) {
+                        return Mono.defer(() -> Mono.just(new PollResponse<>(LongRunningOperationStatus.fromString("FORBIDDEN", true),
+                            pollingContext.getLatestResponse().getValue())));
```

Comment:
Based on the feedback from API review.
this is not supposed to be here.
This logic is supposed to be in the poll operation of beginCreateCertificate.